OpenRA.Mods.TS/Player/PlaceSimpleBeacon.cs

```diff
@@ -0,0 +1,81 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.TS.Effects;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.TS.Traits
+{
+	[Desc("A beacon that consists of a single sprite that can be animated.")]
+	public class PlaceSimpleBeaconInfo : ITraitInfo
+	{
+		public readonly int Duration = 30 * 25;
+		public readonly string NotificationType = "Sounds";
+		public readonly string Notification = "Beacon";
+
+		public readonly bool IsPlayerPalette = false;
+		[PaletteReference("IsPlayerPalette")] public readonly string Palette = "effect";
+
+		public readonly string BeaconCollection = "beacon";
```

Comment:
@pchote recently mentioned that the term `Collection` should only be used for UI.

I think `BeaconImage` would be better here and more consistent with other traits.